app/views/user_mailer/notify_reset_password_disallowed_due_to_suspension.text.erb

```diff
@@ -5,4 +5,4 @@ Your <%= t('department.name') %> Signon <%= account_name %>, for <%= @user.email
 <%= account_name.humanize %> suspensions do not suspend production accounts.
 <% end %>
 
-If you believe your account has been suspended in error, please contact a managing <%= t('department.name') %> editor in your organisation (or your parent organisation). They can use the support form to get help from the <%= t('department.name') %> team.
+If you believe your account has been suspended in error, please contact a managing <%= t('department.name') %> editor in your organisation (or your parent organisation). They can use the support form (<%= t('support.url') %>) to get help from the <%= t('department.name') %> team.
```

Comment:
Should this be wrapped in an `<a>`?

Reply:
Is that how text is rendered? I couldn't find any clear definition for it. Other links in text.erb don't have the <a> e.g. `<%= link_to t('department.name'), t('department.url') %>` in signon/app/views/user_mailer/notify_reset_password_disallowed_due_to_unaccepted_invitation.text.erb